gclient/src/api/calls.rs

```diff
@@ -222,7 +305,13 @@ impl GearApi {
         }
     }
 
-    /// `pallet_gear::reset`
+    /// Clear data from all pallet storages.
```

Comment:
Note that as reset extrinsic, it exists only before real production.

Reply:
Done